src/foam/nanos/auth/LastModifiedAwareDAO.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * @license
+ * Copyright 2017 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+// TODO: Test
+foam.CLASS({
+  package: 'foam.nanos.auth',
+  name: 'LastModifiedAwareDAO',
+  extends: 'foam.dao.ProxyDAO',
```

Comment:
DAODecorator makes modifying the data on the way out easier, but these decorators are only adding behaviour on put(), so I don't think it buys anything here.